src/IpPort.cpp

```diff
@@ -0,0 +1,31 @@
+#include "IpPort.h"
+
+namespace visor {
```

Comment:
how about a sub namespace for `network`? so this would be `visor::network::IpPort` ... we may move things here over time out of pcapplusplus.

Reply:
done